src/content-handlers/iiif/modules/uv-resourcesleftpanel-module/ResourcesLeftPanel.ts

```diff
@@ -161,7 +162,7 @@ export class ResourcesLeftPanel extends LeftPanel<ResourcesLeftPanelConfig> {
 
   resize(): void {
     super.resize();
-
+    console.log('resources left panel resized')
```

Comment:
Looks like some debug and an unwanted newline got committed here -- presumably unintentionally.